src/main/java/org/team1540/robot2025/Constants.java

```diff
@@ -35,6 +35,6 @@ public enum Mode {
     public static final double ROBOT_MASS_KG = Units.lbsToKilograms(147); // max weight :skull:
     public static final double ROBOT_MOI_KGM2 = 7.74581;
 
-    public static final double BUMPER_LENGTH_X_METERS = Units.inchesToMeters(35.0);
-    public static final double BUMPER_LENGTH_Y_METERS = Units.inchesToMeters(33.0);
+    public static final double BUMPER_LENGTH_X_METERS = Units.inchesToMeters(18.719 * 2);
```

Comment:
?

Reply:
Changed bumper size to actually match what's in CAD